add u-boot.sb binary format in BR menuconfig - Safe Boot (.sb) format
depends on the host-elftosb tool to build requiring a specific dependency
in BuidlRoot makefile

Signed-off-by: Eric Jarrige <eric.jarrige@armadeus.org>
---
--- buildroot/boot/uboot/Config.in
+++ buildroot/boot/uboot/Config.in
@@ -118,6 +118,9 @@ config BR2_TARGET_UBOOT_FORMAT_LDR
 	depends on BR2_bfin
 	bool "u-boot.ldr"
 
+config BR2_TARGET_UBOOT_FORMAT_SB
+	bool "u-boot.sb"
+
 config BR2_TARGET_UBOOT_FORMAT_CUSTOM
 	bool "custom binary format"
 	help
--- buildroot/boot/uboot/uboot.mk
+++ buildroot/boot/uboot/uboot.mk
@@ -37,6 +37,10 @@ else ifeq ($(BR2_TARGET_UBOOT_FORMAT_NAN
 UBOOT_BIN          = u-boot-nand.bin
 else ifeq ($(BR2_TARGET_UBOOT_FORMAT_IMG),y)
 UBOOT_BIN          = u-boot.img
+else ifeq ($(BR2_TARGET_UBOOT_FORMAT_SB),y)
+UBOOT_DEPENDENCIES += host-elftosb
+UBOOT_BIN          = u-boot.sb
+UBOOT_MAKE_TARGET  = $(UBOOT_BIN)
 else ifeq ($(BR2_TARGET_UBOOT_FORMAT_CUSTOM),y)
 UBOOT_BIN          = $(call qstrip,$(BR2_TARGET_UBOOT_FORMAT_CUSTOM_NAME))
 UBOOT_MAKE_TARGET  = $(UBOOT_BIN)
